OMEdit/OMEdit/OMEditGUI/Annotations/LineAnnotation.cpp

```diff
@@ -1315,6 +1315,19 @@ void LineAnnotation::duplicate()
   pLineAnnotation->setSelected(true);
 }
 
+void LineAnnotation::updateConnectionOrTransistionOrInitialState(QString const& annotation)
+{
+  parseShapeAnnotation(annotation);
+  initializeTransformation();
+  removeCornerItems();
+  drawCornerItems();
+  adjustGeometries();
+  setCornerItemsActiveOrPassive();
+  update();
+  emitChanged();
+  updateConnectionAnnotation();
```

Comment:
Remove this line and put it back in `UpdateConnectionCommand::redoInternal()` and `UpdateConnectionCommand::undo()`.

Reply:
Ah I see. Sloppy from my side, I will adjust the other calls